contracts/tasks/storageSlots.js

```diff
@@ -0,0 +1,156 @@
+const path = require('path');
+const { promises, existsSync, mkdirSync } = require('fs');
+
+const {
+  getStorageLayout,
+  getVersion,
+  getUnlinkedBytecode,
+  Manifest,
+  getStorageLayoutForAddress,
+  isCurrentValidationData,
+  assertStorageUpgradeSafe
+} = require('@openzeppelin/upgrades-core');
```

Comment:
Don't we need to add this package to package.json? We currently have @openzeppelin/upgrades in there but not @openzeppelin/upgrades-core

Reply:
I think it worked for me because another package already includes it. Thanks for catching this, added to `contracts/package.json`